src/client/fuse_ll.cc

```diff
@@ -63,6 +65,19 @@
 
 #define _CEPH_CLIENT_ID	"ceph.client_id"
 
+/*
+ * The dedicated struct for snapid <-> stag map for each ceph
+ * inode, and the stag is a number in range [2, 0xffff], and
+ * the stag number 0 is reserved for CEPH_NOSNAP and 1 is
+ * reververd for CEPH_SNAPDIR.
+ */
+struct ceph_fuse_fake_inode_stag {
+  ceph::unordered_map<uint64_t,int> snap_stag_map;
+  ceph::unordered_map<int, uint64_t> stag_snap_map;
+  int last_stag = 1;
+};
+>>>>>>> f9dcf013dc0... ceph-fuse: add dedicated snap stag map for each directory
```

Comment:
looks like this line is a leftover of a merge/rebase conflict resolution

Reply:
Okay, it seems some changes still in the local cache, forgot to amend it. Will fix it.